Chapter5 - Generalizing Gradient Descent - Learning Multiple Weights at a Time.ipynb

```diff
@@ -53,6 +53,7 @@
     "pred = neural_network(input,weights)\n",
     "error = (pred - true) ** 2\n",
     "delta = pred - true\n",
+    "weight_deltas = ele_mul(delta, input)\n",
```

Comment:
weight_deltas needs to come after the ele_mul function, otherwise 'ele_mul' is not defined